src/Sobol.jl

```diff
@@ -1,67 +1,93 @@
 module Sobol
 
-import NLopt.libnlopt
-import Base: ndims, length, skip, next, start, done, show, writemime
+import Base: ndims, skip, next, start, done, show, writemime
 export SobolSeq, ScaledSobolSeq, next!
 
-type SobolSeq{N}
-    p::Ptr{Void} # nlopt_sobol pointer
-    function SobolSeq()
-        (N < 1 || N > 1111) && error("invalid Sobol dimension")
-        p = ccall((:nlopt_sobol_create,libnlopt), Ptr{Void}, (Cuint,), N)
-        p == C_NULL && error("error initializing Sobol sequence")
-        s = new(p)
-        finalizer(s, destroy)
-        s
-    end
+include("soboldata.jl") #loads `sobol_a` and `sobol_minit`
+
+type SobolSeq
+    sdim::Int #dimension of sequence being generated
+    m::Array{UInt,2} #array of size (sdim, WORD_SIZE)
```

Comment:
Should all these `UInt`s be `UInt32` or `UInt64`?   I don't think we want the behavior to depend on `WORD_SIZE`.

`UInt64` allows in principle a longer sequence of numbers, although I'm doubtful that anyone ever needs more than 2^32 points in practice, but it will be probably a bit slower (would be good to benchmark this to see by how much) even on 64-bit machines.